specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-09-02-preview/managedClusters.json

```diff
@@ -6852,6 +6856,21 @@
       "title": "The network mode Azure CNI is configured with.",
       "description": "This cannot be specified if networkPlugin is anything other than 'azure'."
     },
+    "EbpfDataplane": {
+      "type": "string",
+      "enum": ["cilium"],
+      "x-ms-enum": {
+        "name": "ebpfDataplane",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "cilium",
```

Comment:
Is there supposed to be a `none` option here?

Also I just realized this is lowercase and ideally it'd be uppercase `Cilium` to match our other enums. If it's too late to change then carry on as is. I need to review for that more carefully in the future.

Reply:
* aks-rp accepts "none" (treated the same as an empty string), but I left it off because I saw that NetworkPluginMode above didn't have it.
* aks-rp does a case-insensitive compare for the enum string, so upper-case "Cilium" should work. I'll test it then update this PR to use uppercase.

Reply:
Verified that aks-rp also accepts "Cilium" (uppercase). However, the value returned from GET managed cluster is lowercase ("cilium").

Looking at the other networkProfile enums, I see several others that use lowercase enums: for example, NetworkPlugin, NetworkProfile, and NetworkMode.

For these reasons, I'm going to leave cilium as lowercase in this PR for now. Does that sound okay to you @matthchr ?

Reply:
If the RP returns `cilium` then yes, that's probably what should be documented in the Swagger. Also yes, there are other enums that use `lowercase`. IIRC the breakdown is ~27 UpperCamelCase and ~6-7 lowerCamelCase enums in the API all-up though, and I'm trying to match the 27 but only just started reviewing for it, so missed this one and another lowercase enum being added in this API. If the ship has sailed on the RP release so we can't change what RP returns (which I think it has), then leave this as `cilium`.